Tests/Test_SendTritium.c

```diff
@@ -0,0 +1,337 @@
+#include "Minions.h"
+#include "Pedals.h"
+#include "FaultState.h"
+#include "CANbus.h"
+#include "MotorController.h"
+#include "ReadCarCAN.h"
+#include "BSP_UART.h"
+#include "Tasks.h"
+#include "SendTritium.h"
+
+// Inputs
+extern bool cruiseEnable;
+extern bool cruiseSet;
+extern bool regenToggle;
+extern uint8_t brakePedalPercent;
+extern uint8_t accelPedalPercent;
+extern bool reverseSwitch;
+extern bool forwardSwitch;
+extern uint8_t state;
+extern float currentSetpoint;
+extern float velocitySetpoint;
+extern float velocityObserved;
+extern float cruiseVelSetpoint;
+
+static OS_TCB Task1TCB;
+static CPU_STK Task1Stk[DEFAULT_STACK_SIZE];
+
+void stateBuffer(){
+    OS_ERR err;
+    OSTimeDlyHMSM(0, 0, 0, 20, OS_OPT_TIME_HMSM_STRICT, &err);
+    assertOSError(OS_UPDATE_VEL_LOC, err);
+}
+
+void goToBrakeState(){
+    // Brake State
+    brakePedalPercent = 15;
+    stateBuffer();
+}
+
+
+void goToNormalState(){
+    goToBrakeState();
+
+    //Normal State
+    brakePedalPercent = 0;
+    stateBuffer();
+}
+
+
+void goToOnePedalDrive(){
+    goToNormalState();
+
+    // One Pedal Drive
+    cruiseEnable = false;
+    regenToggle = true;
+    ChargeEnable = true;
```

Comment:
I'm guessing the raw ChargeEnable value is used a lot in this file, but it shouldn't be. Assuming you implemented the suggestions i made above in SendTritium, change this to `regen_enable` and it should work pretty much the same way, but we're protecting the value inside of ReadCarCAN properly.